### ISSUE TITLE
examples

### ISSUE THREAD

**Issue by ameensol** [edge/fluxette#12]
The examples that you wrote for your previous build were really helpful in figuring out how all the different pieces of fluxette work together and saved me a great deal of headache. It would be awesome to get some updated examples for the new version so I don't waste time making trivial uninformed errors during the implementation. 

P.S. do you still use the dev[at]ionized email address? I sent you an email a few days ago, wasn't sure if you got it.

**Comment by edge** [comment]
Ah, I'll do that soon. My trip's been pretty busy lately.

**Comment by edge** [comment]
So I've rewritten the dispatcher (yet again). It doesn't have very many external changes, but allows for a more robust middleware and dispatch system. Allowing both async/thunk actions and array dispatches presented a challenge, but I ended up doing it anyways.

The only external change that you'll really see is a return to the more simple `Flux(stores)` factory method. I've removed all of the `Bridge`, `Factory`, and Generic and Bound `Interface` cruft, since in the end, it wasn't ergonomic and didn't align with our priorities.

Otherwise, the `Store` factory has been renamed to `Shape`, because it was kind of misleading.

I've updated the flux-comparison example to reflect the 0.15.x API.
